zcash_client_backend/src/data_api.rs

```diff
@@ -44,6 +47,95 @@ pub enum NullifierQuery {
     All,
 }
 
+#[derive(Debug, Clone)]
+pub struct AccountBalance {
+    pub spendable_value: NonNegativeAmount,
+    pub pending_change_value: NonNegativeAmount,
+    pub value_pending_spendability: NonNegativeAmount,
```

Comment:
Naming is hard. I didn't want to actually expose change as separate from value pending spendability, because I don't think the difference between them is all that important, but the current Zashi wallet designs have them separated out (though recently received non-change notes with fewer than `min_confirmations` and old change notes that are awaiting subtree scanning are not distinguished from one another.)

Reply:
Naming has been improved, unresolve if necessary.